src/java/org/apache/ivy/Main.java

```diff
@@ -199,6 +201,10 @@ static CommandLineParser getParser() {
                     new OptionBuilder("cp").arg("cp")
                             .description("extra classpath to use when launching process").create())
 
+                .addCategory("maven compatibility options")
+                .addOption(new OptionBuilder("pomfile").arg("pomfile").countArgs(false)
+                            .description("makepom as standalone tasks").create())
```

Comment:
I think the description should be a bit more clear and state that this generates a pom file for the resolved module.

Reply:
IMHO `pomfile` is confusing (cf use of `ivyfile` and `propertiesfile`). I'd suggest `writepom`or something like that.

Reply:
On the second thoughts, why not calling the option `makepom` 😉 ?